moving_modules.md

```diff
@@ -0,0 +1,317 @@
+        lib/puppet/type/mailalias.rb
+        lib/puppet/provider/mailalias/aliases.rb
+     
+      maillist_core/
+        lib/puppet/type/maillist.rb
+        lib/puppet/provider/maillist/maillist.rb
+
+      macdslocal_core/                             Depends on 'puppet/provider/nameservice/directory_service',
+        lib/puppet/type/                                      'puppet/util/plist'
+          computer.rb
+          macauthorization.rb
+          mcx.rb
+        lib/puppet/provider/
+          computer.rb
+          macauthorization.rb
+          mcxcontent.rb
+
+      mount_core/                                  Depends on 'puppet/property/boolean',
+        lib/puppet/type/mount.rb                              'puppet/provider/parsedfile'
+        lib/puppet/provider/mount.rb
+        lib/puppet/provider/mount/parsed.rb
+
+      nagios_core/                                 Depends on 'puppet/provider/parsedfile'
+        lib/puppet/external/nagios.rb
+        lib/puppet/external/nagios/
+          base.rb
+          grammer.py
+          makefile
+          parser.rb
+        lib/puppet/type/
+          nagios_*.rb
+        lib/puppet/provider/naginator.rb          
+        lib/puppet/util/nagios_maker.rb
+
+      network_device_core/
+        lib/puppet/feature/telnet.rb
+        lib/puppet/type/
+          router.rb
+          interface.rb
+          vlan.rb
+        lib/puppet/provider/
+          cisco.rb
+          interface/cisco.rb
+          vlan/cisco.rb
+        lib/puppet/util/
+          network_device.rb
+          network_device/*
+
+      scheduled_task_core/                         Depends on 'puppet/util/windows'
+        lib/puppet/type/scheduled_task.rb
+        lib/puppet/provider/scheduled_task/win32_taskscheduler.rb
+        lib/puppet/util/windows/taskscheduler.rb
+      
+      selinux_core/                                Depends on 'puppet/type/file/selcontext',
+        lib/puppet/feature/selinux.rb                         'puppet/util/selinux'
+        lib/puppet/type/
+          selboolean.rb
+          selmodule.rb
+        lib/puppet/provider/
+          selmodule/seboolean.rb
+          selmodule/semodule.rb
+
+      sshkeys_core/                                Depends on 'puppet/provider/parsed'
+        lib/puppet/type/
+         sshkey.rb
+         ssh_authorized_key.rb
+        lib/puppet/provider/
+         sshkey/parsed.rb
+         ssh_authorized_key/parsed.rb
+
+      yumrepo_core/                                Depends on 'puppet/util/filetype'
+        lib/puppet/type/yumrepo.rb
+        lib/puppet/provider/yumrepo/inifile.rb
+        lib/puppet/util/inifile.rb
+
+      zfs_core/
+        lib/puppet/type/zfs.rb
+        lib/puppet/provider/zfs/zfs.rb
+
+      zone_core/                                   Depends on 'puppet/property/list'
+        lib/puppet/type/zone.rb
+        lib/puppet/provider/zone/zone.rb
+
+      zpool_core/
```

Comment:
This is being rolled into the zfs module.

Reply:
updated